packages/hooks/src/useElementNode.ts

```diff
@@ -0,0 +1,16 @@
+import { useState, useCallback } from 'react';
+
+/**
+ * Hook to subscribe to changes in a ref element.
```

Comment:
Let's include an example of how to use this in these comments, cuz this hook is a little hard to understand by just looking at it.

Reply:
👍

Reply:
As a note, let's include an example by default in our tsdoc annotations.